server/main.go

```diff
@@ -13,8 +14,10 @@ import (
 )
 
 func main() {
-	configfn := "default.conf"
-	data, err := ioutil.ReadFile(configfn)
+	configfn := flag.String("config", "default.conf", "location of configuration file")
```

Comment:
I think we use cobra framework because it faciliates the way to write subcommand. Considering we don't use subcommand in server side, it is fine to use standard flag lib here.